calculator/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <title>Calculator App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="calc.css">
+    <link rel="icon" href="img/redi.png">
+</head>
+<body>
+<div class="headerBar">
+    <ul class="headerBar__links">
+        <li><a class="headerBar__link" href="/">Home</a></li>
```

Comment:
Yeah I think this is because you are serving the site incorrectly on local. It should be served through a web server to imitate the functionality of git pages.

Reply:
I believe we have not yet instructed students on how to serve the app locally.

Reply:
Its in the setup video. And I've seen students using it in my session